plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -161,3 +162,15 @@ export enum MongodbCommandType {
   COUNT = 'count',
   UNKNOWN = 'unknown',
 }
+
+// https://github.com/mongodb/node-mongodb-native/blob/v4.2.2/src/cmap/connection.ts
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export class V4Connection {
```

Comment:
I think this should just be an interface or a type. Its concrete implementation is never used.

Reply:
Agree! fixed